domain/machine/errors/errors.go

```diff
@@ -22,4 +22,8 @@ const (
 
 	// InvalidStatus describes a status that is not valid
 	InvalidStatus = errors.ConstError("invalid status")
+
+	// GrandParentNotSupported describes an error that occurs when the operation
+	// found a grandparent machine, as it is not currently supported.
+	GrandParentNotSupported = errors.ConstError("grandparent machine are not supported currently")
```

Comment:
"machines"

Reply:
Oups. A bit late.

Would have push a typo PR, but it will mess up with the @cderici incoming PR: https://github.com/juju/juju/pull/17759

@cderici maybe you can fix this typo after rebase ?